src/challah.py

```diff
@@ -72,10 +77,10 @@ def forward(self, x):
         x = x.view(batch_size, -1, 2)
         x = self.rasterize(x)
         x = x.unsqueeze(1)
-        x = self.resnet(x)
+        x = self.densenet(x)
```

Comment:
This file _replaces_ `eggtart.py`. You should just add another module, so both pastries exist side-by-side.

Reply:
This means you'll also have to put `resnet.py` back; you've deleted it.